# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: image/imgLoader.cpp
# Commit: 75b04de7e99c
# Full Hash: 75b04de7e99cab7bed5cc8b0f2f6d52ddf174bf3
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-25 09:39:13
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r 91c3acdb2454 -r 75b04de7e99c image/imgLoader.cpp
--- a/image/imgLoader.cpp	Fri May 24 11:32:20 2019 +0000
+++ b/image/imgLoader.cpp	Fri May 24 11:32:54 2019 +0000
@@ -15,6 +15,7 @@
 #include "mozilla/Move.h"
 #include "mozilla/NullPrincipal.h"
 #include "mozilla/Preferences.h"
+#include "mozilla/StaticPrefs.h"
 #include "mozilla/ChaosMode.h"
 #include "mozilla/LoadInfo.h"
 #include "mozilla/StaticPrefs.h"
@@ -55,7 +56,6 @@
 #include "nsIMemoryReporter.h"
 #include "DecoderFactory.h"
 #include "Image.h"
-#include "gfxPrefs.h"
 #include "prtime.h"
 #include "ReferrerInfo.h"
 
@@ -1251,8 +1251,8 @@
 }
 
 void imgLoader::GlobalInit() {
-  sCacheTimeWeight = gfxPrefs::ImageCacheTimeWeight() / 1000.0;
-  int32_t cachesize = gfxPrefs::ImageCacheSize();
+  sCacheTimeWeight = StaticPrefs::ImageCacheTimeWeight() / 1000.0;
+  int32_t cachesize = StaticPrefs::ImageCacheSize();
   sCacheMaxSize = cachesize > 0 ? cachesize : 0;
 
   sMemReporter = new imgMemoryReporter();